lib/openstax/rescue_from/mute_listener.rb

```diff
@@ -0,0 +1,19 @@
+module OpenStax
+  module RescueFrom
+    class MuteListener
```

Comment:
Oh ya it was used and now it's not. It could be used again if I move the guts between the hooks into a service class, as the service class would take the controller as its listener